.github/workflows/pytest-and-pylint.yml

```diff
@@ -0,0 +1,44 @@
+name: Run pytest and pylint
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run_pylint:
+    name: Run pylint
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -el {0}
```

Comment:
I'm curious why we need this line. Can it be removed?

Reply:
No, you may not